apps/meteor/definition/voip/SignallingSocketEvents.ts

```diff
@@ -0,0 +1,8 @@
+export type SignallinSocketEvents = {
+	connected: undefined;
+	disconnected: undefined;
+	connectionerror: unknown;
+	localnetworkonline: undefined;
+	localnetworkoffline: undefined;
+};
+export type SocketEventKeys = keyof SignallinSocketEvents;
```

Comment:
this file seems duplicated from `packages/core-typings/src/voip/SignallingSocketEvents.ts`